skyportal/tests/issues/test_handlers_issues.py

```diff
@@ -0,0 +1,179 @@
+from skyportal.handlers.api.stream import StreamHandler
+from skyportal.handlers.api.filter import FilterHandler
+from skyportal.handlers.api.candidate import CandidateHandler
+from skyportal.handlers.api.photometry import PhotometryHandler
+from urllib import request
+import json
+from tornado import httputil
+
+
+stream_id = None
+
+class FakeApplication:
+    """Mimics the Application class of tornado.web.
+    Subclasses were minimum required to import the class.
+    """
+
+    def __init__(self,) -> None:
+        super().__init__()
+        self.ui_methods = {}
+        self.ui_modules = {}
+        self.settings = {}
+
+
+
+class FakeRequest(httputil.HTTPServerRequest):
+    """Mimics the HTTP request handler of tornado.web.
+    Subclasses were minimum required to import the class.
+    """
+
+    def __init__(
+        self,
+    ):
+        self.test = True
+        
+        super().__init__()
+
+        class FakeConnection(object):
+            """Mimics the connection within the HTTP request handler."""
+
+            def __init__(
+                self,
+            ):
+                def set_close_callback(self):
+                    return True
+
+                self.set_close_callback = set_close_callback
+
+
+
+        self.connection = FakeConnection()
+
+application = FakeApplication()
+fake_request = FakeRequest()
+
+# In this test, we post a new Candidate using the CandidateHandler
+def test_candidate_handler(super_admin_user):
+    #FILTER
+    filter_handler = FilterHandler(
+        application=application,
+        request=fake_request,
+    )
+    filter_handler.current_user = super_admin_user
+    try:
+        filter_handler.get()
+    except: None
+    filter_id = json.loads(filter_handler._write_buffer[0].decode('utf-8'))['data'][0]['id']
+    assert filter_id is not None
+    #CANDIDATE   
+    candidate_handler = CandidateHandler(
+        application=application,
+        request=fake_request,
+    )
+
+    body = {
+        'id': 'ZTF_TEST',
+        'filter_ids': [filter_id],
+        'passed_at': 59000,
+        'ra': 10,
+        'dec': 10,    
+    }
+
+    candidate_handler.current_user = super_admin_user
+    candidate_handler.request.body = json.dumps(body, indent=2).encode('utf-8')
+    try:
+        candidate_handler.post()
```

Comment:
What kind of error do you get when you try to post?

Reply:
I have asked @Theodlz to retry the external photometry post once #2648 is in.